### PR TITLE
Init disposable email project

### PR DESCRIPTION
**Pull request by pradzikowski**
I was closed last mail because i made some mess with flating my commits.
So i was rebuild my copy and create new MR.

So i'm waiting for MR:

### CODE REVIEW

File: test/AbstractPCFVerifierTest.php [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    }
+    
+    public function testNotAcceptedStatus(): void
+    {
+        $this->expectException(VerifyDomainException::class);
+        $this->expectExceptionMessage('Validation result status is not accepted. Maybe u used custom result?');
+
+        $collection = new SimpleDomainCollection();
+        $verifier   = new class extends AbstractPCFVerifier {
+            protected function doVerifyDomain(string $domain): int {
```
Comment (adeptofvoltron) [inline test/AbstractPCFVerifierTest.php]: Opening brace should be on a new line

File: test/AbstractPCFVerifierTest.php [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    }
+
+    public function testEmptyStringException(): void
+    {
+        $this->expectException(VerifyDomainException::class);
+        $this->expectExceptionMessage('Inserted domain is empty string');
+
+        $collection = new SimpleDomainCollection();
+        $verifier   = new class extends AbstractPCFVerifier {
+            protected function doVerifyDomain(string $domain): int {
```
Comment (adeptofvoltron) [inline test/AbstractPCFVerifierTest.php]: Opening brace should be on a new line

File: test/AbstractPCFVerifierTest.php [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+class AbstractPCFVerifierTest extends TestCase
+{
+    public function testNotSetDomainCollectionException(): void
+    {
+        $this->expectException(VerifyDomainException::class);
+        $this->expectExceptionMessage('DomainCollection does not set');
+
+        $verifier = new class extends AbstractPCFVerifier {
+            protected function doVerifyDomain(string $domain): int {
```
Comment (adeptofvoltron) [inline test/AbstractPCFVerifierTest.php]: Opening brace should be on a new line

File: README.md [hunk 4]
```diff
@@ -1 +1,39 @@
...
+
+These repository is contributed by [MIT License](LICENSE) but files [BlockList.php](src/Resource/BlockList.php) and [TrustList.php](src/Resource/TrustList.php) are contributed by [Public Domain License(CC0 1.0)](https://creativecommons.org/publicdomain/zero/1.0/).
+
+# Basic Usage
+
+```PHP
+<?php
+use PCF\DisposableEmail\ListedVerifier;
+use PCF\DisposableEmail\SimpleDomainCollection;
+use PCF\DisposableEmail\Resource\BlockList;
```
Comment (adeptofvoltron) [inline README.md]: it not exist anymore

File: README.md [hunk 5]
```diff
@@ -1 +1,39 @@
...
+use PCF\DisposableEmail\Resource\TrustList;
```
Comment (adeptofvoltron) [inline README.md]: it not exist anymore

**Review by adeptofvoltron — CHANGES_REQUESTED** [review]
change the example in README.md to working one

File: test/AbstractPCFVerifierTest.php (hunk 3 above)
Comment (pradzikowski) [reply test/AbstractPCFVerifierTest.php]: done

File: test/AbstractPCFVerifierTest.php (hunk 2 above)
Comment (pradzikowski) [reply test/AbstractPCFVerifierTest.php]: done

File: test/AbstractPCFVerifierTest.php (hunk 1 above)
Comment (pradzikowski) [reply test/AbstractPCFVerifierTest.php]: done

**Comment by pradzikowski** [comment]
@adeptofvoltron done

**Review by adeptofvoltron — APPROVED** [review]

File: test/AbstractPCFVerifierTest.php [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+
+    public function testEmptyStringException(): void
+    {
+        $this->expectException(VerifyDomainException::class);
+        $this->expectExceptionMessage('Inserted domain is empty string');
+
+        $collection = new SimpleDomainCollection();
+        $verifier   = new class extends AbstractPCFVerifier
+        {
+            protected function doVerifyDomain(string $domain): int {
```
Comment (adeptofvoltron) [inline test/AbstractPCFVerifierTest.php:37]: brace

**Review by adeptofvoltron — CHANGES_REQUESTED** [review]
Conflicted

**Pull request merged**